system/CLI/CLI.php

```diff
@@ -272,7 +273,35 @@ public static function prompt(string $field, $options = null, string $validation
 			}
 		}
 
-		return empty($input) ? '' : $input;
+		return (string) $input;
+	}
+
+	/**
+	 * Asks the user to select a specific option out of a set of options based on the option's key
+	 *
+	 * @param string            $text       Output "field" text
+	 * @param array             $options    a list of options (array(key => description)), the first option will be the default value
+	 * @param string|array|null $validation Validation rules
+	 *
+	 * @return string The selected key
+	 *
+	 * @codeCoverageIgnore
+	 */
+	public static function promptSelect(string $text, array $options, $validation = null): string
+	{
+		if (!$options) {
+			throw new InvalidArgumentException('No options to select from provided');
+		}
```

Comment:
```suggestion
		if (! $options)
		{
			throw new InvalidArgumentException('No options to select from were provided');
		}
```

Reply:
Talk about muscle memory.. and dev's preference ^^
Even when I think about it the line before, it always ends up like that.

Wouldn't that be a great job for rector/lexer or is the PR just to old for them to work properly?

Reply:
Our current CICD does not offer yet the code styling checks as we were bumping to problems before. If you have been developing this locally then the precommit hook should do the job. If with GH desktop, I guess that does not work so we have to manually check CS.

Reply:
Precommit hook didn't show it, maybe the branch had old rules since I remember stumbling over this in another branch commit.

Reply:
run `bash admin/setup.sh`

Another CS checks. combine else if into one word. Then ! should have one space the variable being negated.